dashboard/test/models/user_test.rb

```diff
@@ -679,10 +679,25 @@ def assert_fails_email_uniqueness_validation(user)
     end
   end
 
-  test "cannot create teacher without email" do
-    assert_does_not_create(User) do
```

Comment:
is there a specific reason to move away from the `assert_does_not_create` test?

Reply:
yes, but that's because I didn't realize `assert_does_not_create` returns the user object. I have brought this assertion back.